src/adapters/amqp.js

```diff
@@ -499,6 +500,11 @@ class AmqpAdapter extends BaseAdapter {
 		// Adapter is stopping. Publishing no longer is allowed
 		if (this.stopping) return;
 
+		if (!this.connected) {
```

Comment:
I don't think it's a good idea, because it can make data loss if the developer doesn't know that the message is not sent. So better if we throw an error.

Reply:
Done

Reply:
Please use MoleculerRetryableError because this kind of issue will be solved by the time.